cli/tests/test_duplicate_command.rs

```diff
@@ -2427,6 +2427,62 @@ fn test_rebase_duplicates() {
     ");
 }
 
+#[test]
+fn test_duplicate_description_template() {
+    let test_env = TestEnvironment::default();
+    test_env.run_jj_in(".", ["git", "init", "repo"]).success();
+    let work_dir = test_env.work_dir("repo");
+
+    create_commit(&work_dir, "a", &[]);
+    create_commit(&work_dir, "b", &[]);
+    create_commit(&work_dir, "c", &["a", "b"]);
+
+    // Test the setup
+    insta::assert_snapshot!(get_log_output(&work_dir), @r"
+    @    17a00fc21654   c
+    ├─╮
+    │ ○  d370aee184ba   b
+    ○ │  2443ea76b0b1   a
+    ├─╯
+    ◆  000000000000
+    [EOF]
+    ");
+
+    test_env.add_config(r#"templates.duplicate_description = "concat(description, '\n(cherry-picked from commit ', commit_id, ')')""#);
+    let output = work_dir.run_jj(["duplicate", "a"]);
```

Comment:
Can you add test for `duplicate_commits()`?

Reply:
Let me know if this is sufficient.

Reply:
I meant to add CLI test for both `duplicate_commits()` and `duplicate_commits_onto_parents()`, but adding lib tests seems also good.

Reply:
What do you mean add CLI tests for them? `duplicate_commits()` is tested by the case I added (via `jj duplicate`), and I can add one for `duplicate_commits_onto_parents()` if you want.

It seems like you're talking about something different though -- could you please elaborate?

Reply:
`jj duplicate` (with no destination arguments) doesn't exercise the `duplicate_commits()` code path, right? For historical reasons, `jj duplicate` has two different behaviors.

Reply:
Ah, right, so you meant that I currently test `duplicate_commits_onto_parents()`, but not `duplicate_parents()`. Yup, can fix. Sorry for the miscommunication, I thought you were talking about something else entirely.